controllers/deleteComments.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { fetchDeleteComments } = require('../Models/deleteComments.models')
+
+exports.deleteComments = (req, res) => 
+{
+    const { comment_id } = req.params
+
+    return fetchDeleteComments(comment_id).then((deletedComment) => 
+    {
+        res.status(204).send(deletedComment)
```

Comment:
DeletedComment is not gonna be sent back here. When status is 204, Express only responds with the status code.